src/mono/wasm/runtime-test.js

```diff
@@ -49,40 +52,27 @@ if (is_browser) {
 	let consoleWebSocket = new WebSocket(consoleUrl);
 	consoleWebSocket.onopen = function(event) {
 		proxyJson(function (msg) { consoleWebSocket.send (msg); });
-		globalThis.testConsole.log("browser: Console websocket connected.");
+		console.log("browser: Console websocket connected.");
 	};
 	consoleWebSocket.onerror = function(event) {
-		console.log(`websocket error: ${event}`);
+		console.error(`websocket error: ${event}`);
 	};
-
-	// We expect to be run by tests/runtime/run.js which passes in the arguments using http parameters
-	var url = new URL (decodeURI (window.location));
-	arguments = [];
-	for (var v of url.searchParams) {
-		if (v [0] == "arg") {
-			arguments.push (v [1]);
-		}
-	}
 }
-//proxyJson(console.log);
-
-
-let print = globalThis.testConsole.log;
-let printErr = globalThis.testConsole.error;
 
 if (typeof crypto === 'undefined') {
 	// **NOTE** this is a simple insecure polyfill for testing purposes only
 	// /dev/random doesn't work on js shells, so define our own
 	// See library_fs.js:createDefaultDevices ()
-	var crypto = {
+	crypto = {
 		getRandomValues: function (buffer) {
```

Comment:
We probably need a todo item (like an open issue against dotnet/runtime) tracking the need to implement this properly for the node environment, once this PR lands